pom.xml

```diff
@@ -34,7 +34,7 @@
             http://junit.org/junit5/docs/current/user-guide/#running-tests-build-maven
```

Comment:
If you confirm it works with `2.21.0`, can you remove the comment as well ? I've just setup Travis CI, it will re-trigger a build and check everything :)

Reply:
Done